src/Forms/Forms.js

```diff
@@ -227,76 +228,95 @@ FormSelect.propTypes = {
 };
 
 // ------------------------------------------------- Form Radio ----------------------------------------------
-export class FormRadio extends Component {
+export class FormRadioGroup extends Component {
     constructor(props) {
         super(props);
 
-        this.state = {
-            selectedItem: this.props.defaultChecked
-        };
+        this.groupId = shortId.generate();
     }
 
-    handleChange = e => {
-        this.setState({
-            selectedItem: e.currentTarget.value
-        });
-    };
-
     render() {
-        const { inputs, disabled, isInline } = this.props;
-        let result;
-
-        if (isInline) {
-            result = inputs.map(inputItem => (
-                <div
-                    className='fd-form__item fd-form__item--inline fd-form__item--check'
-                    key={inputItem.id}>
-                    <label className='fd-form__label' htmlFor={inputItem.id}>
-                        <input
-                            checked={this.state.selectedItem === inputItem.id}
-                            className='fd-form__control'
-                            disabled={disabled}
-                            id={inputItem.id}
-                            name={inputItem.name}
-                            onChange={this.handleChange}
-                            type='radio'
-                            value={inputItem.value} />
-                        {inputItem.label}
-                    </label>
-                </div>
-            ));
-        } else {
-            result = inputs.map(inputItem => (
-                <div className='fd-form__item fd-form__item--check' key={inputItem.id}>
-                    <input
-                        checked={this.state.selectedItem === inputItem.id}
-                        className='fd-form__control'
-                        disabled={disabled}
-                        id={inputItem.id}
-                        name={inputItem.name}
-                        onChange={this.handleChange}
-                        type='radio'
-                        value={inputItem.value} />
-                    <label className='fd-form__label' htmlFor={inputItem.id}>
-                        {inputItem.label}
-                    </label>
-                </div>
-            ));
-        }
-        return <div>{result}</div>;
+        const { children, disabled, isInline, onChange, ...props } = this.props;
+
```

Comment:
Should this also have classnames so we can pass a custom one if needed?

Reply:
This should be handled by the `...props` spread. 😄